src/extension/background-script/actions/lnurl/auth.ts

```diff
@@ -94,6 +97,14 @@ export async function authFunction({
         authResponse?.data?.reason || "Auth: Something went wrong"
       );
     } else {
+      const dbAlbyEvent: DbAlbyEvent = {
```

Comment:
I think for such things the pubsub idea was there. 

several subscribes can subscribe to an event like "logged in".